staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -60,6 +60,7 @@ type PodDisruptionBudgetStatus struct {
 	// the list automatically by PodDisruptionBudget controller after some time.
 	// If everything goes smooth this map should be empty for the most of the time.
 	// Large number of entries in the map may indicate problems with pod deletions.
+	// +optional
 	DisruptedPods map[string]metav1.Time `json:"disruptedPods" protobuf:"bytes,2,rep,name=disruptedPods"`
```

Comment:
if this is optional, it should also set omitempty. that would mean this field would no longer be serialized as `disruptedPods: null` and would instead be omitted entirely.